lib/core/entities/artist_entity/artist_entity.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+import 'package:flutter/foundation.dart';
+import '../music_entity/music_entity.dart';
+part 'artist_entity.freezed.dart';
+
+@freezed
+class Artist with _$Artist {
+  factory Artist({
+    required String id,
+    required String name,
+    required String about,
+    required String? socialNetwork,
```

Comment:
lets move this to a list of strings for the moment could be multiple social networks, also probably later we want to pass this to an object or a map of Map<String,String> "social_network name", "url"

Reply:
Fixed.